lib/routes/jrj/index.ts

```diff
@@ -0,0 +1,127 @@
+import { Route } from '@/types';
+
+import cache from '@/utils/cache';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import { ofetch } from 'ofetch';
+
+const options = {
+    '103': '财经资讯',
+    '508': '科技资讯',
+    '106': '商业资讯',
+    '632': '消费资讯',
+    '630': '医疗资讯',
+    '119': '康养资讯',
+    '004': '汽车资讯',
+    '009': '房产资讯',
+    '629': 'ESG 资讯',
+    '010': 'A股资讯',
+    '001': '港股资讯',
+    '102': '美股资讯',
+    '113': '银行资讯',
+    '115': '保险资讯',
+    '104': '基金资讯',
+    '503': '私募资讯',
+    '112': '信托资讯',
+    '007': '外汇资讯',
+    '107': '期货资讯',
+    '118': '债券资讯',
+    '603': '券商资讯',
+    '105': '观点',
+};
+
+export const route: Route = {
+    path: '/:channelNum',
+    categories: ['finance'],
+    example: '/finance',
+    parameters: {
+        column: {
```

Comment:
This parameter is not defined in https://github.com/DIYgod/RSSHub/blob/b1415b090f536aa74d12faaaa935a3a2ef188104/lib/routes/jrj/index.ts#L34